documentation/modules/deploying/proc-deploy-migrate-kraft.adoc

```diff
@@ -0,0 +1,256 @@
+// Module included in the following assemblies:
+//
+// deploying/deploying.adoc
+
+[id='proc-deploy-migrate-kraft-{context}']
+= Migrating to KRaft mode
+
+[role="_abstract"]
+If you are using ZooKeeper for metadata management in your Kafka cluster, you can migrate to using Kafka in KRaft mode. 
+KRaft mode replaces ZooKeeper for distributed coordination, offering enhanced reliability, scalability, and throughput.
+
+During the migration, you install a quorum of controller nodes as a node pool, which replaces ZooKeeper for management of your cluster. 
+You enable KRaft migration in the cluster configuration by applying the `strimzi.io/kraft: migration` annotation.  
+After the migration is complete, you switch the brokers to using KRaft and the controllers out of migration mode using the `strimzi.io/kraft: enabled` annotation.
+
+Before starting the migration, verify that your environment can support Kafka in KRaft mode, as there are a number of xref:ref-operator-use-kraft-feature-gate-str[limitations].
+Note also, the following:
+
+* Migration is only supported on dedicated controller nodes, not on nodes with dual roles as brokers and controllers.
+* Throughout the migration process, ZooKeeper and Controller nodes operate in parallel for a period, requiring sufficient compute resources in the cluster.
+
+.Prerequisites
+
+* You must be using Strimzi 0.40 or newer with Kafka 3.7.0 or newer. If you using an earlier version of Strimzi or Apache Kafka, upgrade before migrating to KRaft mode.
+* Verify that the ZooKeeper-based deployment is operating without the following, as they are not supported in KRaft mode:
+** The Topic Operator running in bidirectional mode. It should either be in unidirectional mode or disabled.
+** JBOD storage. While the `jbod` storage type can be used, the JBOD array must contain only one disk.
+* The Cluster Operator that manages the Kafka cluster is running.
+* The Kafka cluster deployment uses Kafka node pools.
++
+If your ZooKeeper-based cluster is already using node pools, it is ready to migrate.
+If not, you can xref:proc-migrating-clusters-node-pools-str[migrate the cluster to use node pools]. 
+To migrate, brokers must be contained in a `KafkaNodePool` resource configuration that is assigned a `broker` role and has the name `kafka`.
```

Comment:
Why? The `kafka` name is true if you are migrating from NO node pools to node pools first (by following the link we provide on the previous line) but if your cluster was created with node pools since the beginning, the broker pools could be also named "foo", "bar", whatever.

Reply:
I've added the bold to make it clear:

> If your ZooKeeper-based cluster is already using node pools, it is ready to migrate.
> If not, you can xref:proc-migrating-clusters-node-pools-str[migrate the cluster to use node pools]. 
> To migrate **when the cluster is not using node pools**, brokers must be contained in a `KafkaNodePool` resource configuration that is assigned a `broker` role and has the name `kafka`.
> Support for node pools is enabled in the `Kafka` resource configuration using the `strimzi.io/node-pools: enabled` annotation.